### ISSUE TITLE
flash plugin error in nw13

### ISSUE THREAD

**Issue by wzquyin** [nwjs/nw.js#4330]
package.json:
{
...
	 "webkit": {
    "plugin": true
  }
..
}

i copy NPSWF64_20_0_0_286.dl to appfloder /plugins

in nw12 ,it is ok
but in nw 13,show :"adobe flash player is out of date"

ps:flash 20.286 is the lastest version,.......

**Comment by rogerwang** [comment]
You should use Pepper based flash plugin. upstream drops support for NPAPI based plugin....

**Comment by rogerwang** [comment]
PS, try to put pepper flash plugin in 'PepperFlash' directory.

**Comment by srojales** [comment]
hi Roger,

am using NWJS 0.14.3 version, I put C:\Program Files (x86)\Google\Chrome\Application\50.0.2661.94\PepperFlash inside the nwjs app bundle directory, run the app but still cannot load flash from my app particularly the flash in an iframe. 

I also tried putting the content of PepperFlash folder (manifest.json, pepflashplayer.dll) to plugins same level as PepperFlash / nw.exe

Still flash does not work.

I unisntalled flash players in my machines to test.

Is there changes in version 0.14 that prevents flash from running?

OR is there something  I missed?

Thanks roger.
